onnx/defs/experiments/experiments_functions.cc

```diff
@@ -171,5 +171,11 @@ static Common::Status BuildMVN(std::unique_ptr<FunctionProto>* func_proto) {
   return Common::Status::OK();
 }
 
-ONNX_FUNCTION(
+ONNX_FUNCTION_BUILD(
+    MeanVarianceNormalization,
+    9,
     FunctionBuilder().SetDomain(ONNX_DOMAIN).SetBuildFunction(BuildMVN));
+
+} // namespace ONNX_NAMESPACE
+// ONNX_FUNCTION(
```

Comment:
clean up?

Reply:
This means to leave a example for registering functions. I ll update the marco to be the latest one.